python/tvm/script/intrin.py

```diff
@@ -180,6 +185,11 @@ def opaque_axis(begin, end, span):
     return get_axis(begin, end, "opaque", span)
 
 
+@register
+def Select(cond, if_body, else_body, span):  # pylint: disable=invalid-name
```

Comment:
Could you please explain why you use capital `S` here?

Reply:
I'm using the capital S because it matches the definition in tir (`tir.Select`). The reason we do this is so that people's editors do not complain about undefined functions and so that go to definition still works.

Reply:
I recall that pylint would complain `unexpected-keyword-arg` if arguments mismatch occurs between TVM script and TIR node creation, for example:

```python
tir.if_then_else(1 <= h and h < 15 and 1 <= w and w < 15,
                 A[n, h - 1, w - 1, i, nn, ii],
                 tir.float16(0),
                 dtype="float16")
^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
pylint(unexpected-keyword-arg): Unexpected keyword argument 'dtype' in function call
```

Reply:
Thanks for the explanation. But I still personally prefer `select` since most of the existing Python APIs are using lower case (e.g. `tir.min`, `tir. if_then_else`)

On the other side, `Select` looks OK for me if you still persist on it.

Reply:
@tqchen What do you think?